docker-compose.yml

```diff
@@ -120,34 +120,67 @@ services:
     # nodered opens ports on influxdb so it needs to be able to talk to it.
     links:
       - influxdb
+      - postfix
 
   influxdb:
     restart: unless-stopped
-    image: influxdb:1.2.4
+    image: influxdb:latest
     volumes:
       - "${TTN_DASHBOARD_DATA}influxdb:/var/lib/influxdb"
     environment:
       INFLUXDB_INIT_PWD: "${TTN_DASHBOARD_INFLUXDB_ADMIN_PASSWORD:-!notset}"
       PRE_CREATE_DB: "${TTN_DASHBOARD_INFLUXDB_INITIAL_DATABASE_NAME:-demo}"
+      INFLUXDB_BIND_ADDRESS: "influxdb:8088"
```

Comment:
What is this used for? A comment would be good.

Reply:
--What is this used for?
        Enabling remote backup through Environment variable for extra instance( influxdb-backup ). 

--A comment would be good.
        yes. will add it.